build.freeze.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
This take the test_requirements.txt and requiements.txt files and produces a locked file of all deps. 

This allows you to have reproducible builds and for the github security tooling to alert you of bad deps.